models/endoscopy_inbody_classification/configs/inference.json

```diff
@@ -0,0 +1,117 @@
+{
+    "imports": [
+        "$import json",
+        "$import os",
+        "$import torch"
+    ],
+    "bundle_root": "/workspace/bundle/endoscopy_inbody_classification",
+    "output_dir": "$@bundle_root + '/eval'",
+    "dataset_dir": "/workspace/data/endoscopy_inbody_classification",
+    "test_json": "$@dataset_dir+'/test.json'",
+    "test_fp": "$open(@test_json,'r', encoding='utf8')",
+    "test_dict": "$json.load(@test_fp)",
+    "test_close": "$@test_fp.close()",
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "network_def": {
+        "_target_": "SEResNet50",
+        "spatial_dims": 2,
+        "in_channels": 3,
+        "num_classes": 2
+    },
+    "network": "$@network_def.to(@device)",
+    "preprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "LoadImaged",
+                "keys": "image"
+            },
+            {
+                "_target_": "ToTensord",
+                "keys": "label"
```

Comment:
What label is doing here?

Reply:
Hi @SachidanandAlle ,
      Thank you for your advice. There is no need of this process. I have deleted it.
Thanks,
BIn